streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -450,6 +425,36 @@ public void close(boolean clean) {
         }
     }
 
+        /**
+         * <pre>
+         * - {@link #suspend(boolean) suspend(clean)}
+         *   - close topology
+         *   - if (clean) {@link #commit()}
+         *     - flush state and producer
+         *     - commit offsets
+         * - close state
+         *   - if (clean) write checkpoint
+         * - if (eos) close producer
+         * </pre>
+         * @param clean shut down cleanly (ie, incl. flush and commit) if {@code true} --
+         *              otherwise, just close open resources
+         */
+    @Override
+    public void close(boolean clean) {
```

Comment:
We added 1 line to this right? I don't know why the diff shows such a large change...